mozmeao/django-allow-cidr
# Spring clean + move to Github Actions

This changeset refreshes things a little.

* drops support for end-of-life Python and Django versions
   * refactors to stop using `django.utils.deprecation.MiddlewareMixin`, which is unnecessary with LTS and latest (4.0) Django
* moves CI to Github Actions

### Follow-up actions:
- [ ] A new release of this new `0.4.0` version should be done on PyPi


### To test 
- Check Actions/CI
- Try installing it in a local build of something that uses it
